### PR TITLE
Make heartbeat message self-descriptive

### LINKED ISSUE #1116: Report missing local taker daemon after losing connection to maker
We are briefly displaying the `Please ensure taker daemon is running` alert when we lose connection to the _maker_.

### PR DESCRIPTION
**Pull request by luckysori**
Closes #1116.

Actually has nothing to do with losing connection to the maker specifically. My theory is that the change in the maker status means that delivering the next heartbeat is delayed by a little bit.

### CODE REVIEW

File: taker-frontend/src/useEventSource.ts [hunk 1]
```diff
@@ -18,14 +18,14 @@ export function useEventSource(url: string): [EventSource | null, boolean] {
...
-        }, 5000);
+        }, 10000);
 
         const heartbeatHandler = () => {
             clearTimeout(timer);
             setIsConnected(true);
             timer = setTimeout(() => {
                 setIsConnected(false);
-            }, 5000);
+            }, 10000);
```
Comment (luckysori) [inline taker-frontend/src/useEventSource.ts]: 🤔 We could add a `next_heartbeat` field to the heartbeat message and use that value + a buffer here. That way this value can never go out of sync with the backend.

Comment (thomaseizinger) [reply taker-frontend/src/useEventSource.ts]: Nit: `next` would imply an absolute value? Just `interval` (or `pulse`? 😁) perhaps, otherwise concept ACK.

**Review by thomaseizinger — APPROVED** [review]

**Review by thomaseizinger — APPROVED** [review]

**Review by rishflab — APPROVED** [review]

**Comment by luckysori** [comment]
bors retry

**Pull request merged**